### PR TITLE
Site: Add Alibaba MaxCompute to powered-by page

### PR DESCRIPTION
**Pull request by hsyuan**
The `powered-by.png`'s `Used by` column doesn't have enough space for Alibaba's logo. Need help to put logo on it.

### CODE REVIEW

**Comment by julianhyde** [comment]
I'll do the logo. I generate it from a private google doc.

It would be helpful to include the text 'MaxCompute' along with the Alibaba logo. Do you know the name of the font used in the Alibaba logo?

**Comment by hsyuan** [comment]
Thanks!
I guess the font is `Helvetica`.
Here is a list of Alibaba's logo with different style: https://www.alibabagroup.com/en/news/library

File: site/_docs/powered_by.md [hunk 1]
```diff
@@ -112,6 +112,12 @@ is a distributed JVM-Embeddable Database built on top of
 <a href="https://bookkeeper.apache.org/">Apache BookKeeper</a>.
 It uses Calcite as its SQL Planner.
 
+### MaxCompute
+
+<a href="https://www.alibabacloud.com">Alibaba</a>'s large-scale computing and storage system
```
Comment (xndai) [inline site/_docs/powered_by.md]: "Alibaba's big data computing and storage platform"

Comment (hsyuan) [reply site/_docs/powered_by.md]: updated.

**Pull request closed without merging**